bin/build-coal-image

```diff
@@ -73,8 +73,8 @@ exec > >(tee "${LOGFILE}") 2>&1
 . "${ROOT}/buildtools/lib/trace_logger.sh"
 
 BUILD_TGZ=$(build_spec build-tgz)
-USB_SIZE="4gb"
-USB_SKIP=7812500
+USB_SIZE="8gb"
+USB_SKIP=15625000
```

Comment:
This is in bytes, so we're upping it from 7.8MB to 15.6MB. That's probably an unnecessary change, but I also don't think it hurts anything.

Reply:
I'd rather be safe than sorry here, but I can rebuild with it set to 7812500 quickly enough to see.

Reply:
Reverted.

Reply:
Tested output image, and we're good there.